tests/test_vis.py

```diff
@@ -225,3 +227,29 @@ def test_plot_nades(self):
                 position_transform("de_ancient", -163.84375, "y"),
                 color="red",
             )
+
+    def test_plot_round_map_control(self):
+        """Test plot_round_map_control."""
+        fake_alive_player = {
+            "x": -42.51047897338867,
+            "y": 868.4791870117188,
+            "z": 54.92256546020508,
+            "isAlive": True,
+        }
+        fake_frame = {
+            "t": {"players": [fake_alive_player.copy()] * 5},
+            "ct": {"players": [fake_alive_player.copy()]},
+        }
+
+        round_length = 50
+        test_round = {"frames": [fake_frame] * round_length}
+
+        test_filename = "map_control_test.gif"
+
+        bool_returned = plot_round_map_control(
+            test_filename, "de_inferno", test_round, plot_type="players"
+        )
+
+        assert bool_returned
+        assert os.path.isdir(AWPY_TMP_FOLDER)
+        assert len(os.listdir(AWPY_TMP_FOLDER)) > 0
```

Comment:
is there any possibility that AWPY_TMP_FOLDER has something else in it? perhaps a better test here would be to test if the specific output files exist and if there are not 0 bytes

Reply:
Yeah this is a good point. I'll expand the testing so it checks that each saved frame file exists and checks the filesize > 0 bytes. Addressed in next commit, thanks!